### PR TITLE
Update minikube.md

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
modify "the minikubeVM" to "the minikube VM"

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5161)

### CODE REVIEW

File: docs/getting-started-guides/minikube.md [hunk 1]
```diff
@@ -267,7 +267,7 @@ We recommend you use `ImagePullSecrets`, but if you would like to configure acce
 
 ## Add-ons
 
-In order to have minikube properly start/restart custom addons, place the addons you wish to be launched with minikube in the `~/.minikube/addons` directory.  Addons in this folder will be moved to the minikubeVM and launched each time minikube is started/restarted.
+In order to have minikube properly start/restart custom addons, place the addons you wish to be launched with minikube in the `~/.minikube/addons` directory.  Addons in this folder will be moved to the minikube VM and launched each time minikube is started/restarted.
```
Comment (radhikapc) [inline docs/getting-started-guides/minikube.md]: Remove the extra space before "Addons in this ...". 

I would rewrite it as

In order to have minikube properly start or restart custom addons, place the addons you wish to be launched with minikube in the `~/.minikube/addons` directory. Addons in this folder will be moved to the minikube VM and launched each time minikube is started or restarted.

**Comment by zhangxiaoyu-zidif** [comment]
Hi, @hanxiaoshuai 
could you fix the PR?

**Comment by steveperry-53** [comment]
ping @hanxiaoshuai

**Comment by steveperry-53** [comment]
We have a backlog of PRs that we are trying to get through. So sorry about being a pest, but in the interest of moving forward, ping @hanxiaoshuai.

**Comment by steveperry-53** [comment]
No reply from author since August. Duplicates #5793.

**Pull request closed without merging**